Build failure: cutter

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *cutter*

## Build log
see https://hydra.nixos.org/build/279519010

<details>

<summary>Build Log</summary>

```
/build/source/src/common/PythonManager.cpp: In member function 'void PythonManager::shutdown()':
/build/source/src/common/PythonManager.cpp:136:28: error: 'instance' is not a member of 'PySide::SignalManager'
  136 |     PySide::SignalManager::instance().clear();
      |                            ^~~~~~~~
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->


@Mic92 @dtzWill 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/tools/analysis/rizin/cutter.nix
+++ b/pkgs/development/tools/analysis/rizin/cutter.nix
@@ -1,5 +1,6 @@
 { lib
 , fetchFromGitHub
+, fetchpatch
 , stdenv
 # for passthru.plugins
 , pkgs
@@ -31,6 +32,14 @@ let cutter = stdenv.mkDerivation rec {
     hash = "sha256-TSEi1mXVvvaGo4koo3EnN/veXPUHF747g+gifnl4IDQ=";
     fetchSubmodules = true;
   };
+
+  patches = [
+    # https://github.com/rizinorg/cutter/issues/3384
+    (fetchpatch {
+      url = "https://gitlab.archlinux.org/archlinux/packaging/packages/rz-cutter/-/raw/f736a5709c0b4711760f8242fa77eeaf178c0302/pyside-6.8.patch";
+      hash = "sha256-k1Bn6tCNkbE9r5QLfJTBg1zZZU9R7fG1tyfPgSJyQgg=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

